docs/quick-start/adapters.md

```diff
@@ -3,67 +3,149 @@ section: quick-start
 title: Database setup
 [meta]-->
 
-# Database setup
+# Database Setup
 
-<!-- TODO: Needs some introductory text -->
+Before starting your Keystone project you need to have a database set up and ready for Keystone to use.
 
-## Choosing an adapter
+## Choosing A Database
 
-Keystone currently provides two adapters for connecting to either a MongoDB or PostgreSQL database.
-Choose the [Mongoose adapter](/packages/adapter-mongoose/README.md) for MongoDB or the [Knex adapter](/packages/adapter-knex/README.md) for PostgreSQL.
+Keystone currently provides support for [MongoDB](https://www.mongodb.com/) or [PostgreSQL](https://www.postgresql.org/) databases.
+You will need to pick one of these databases to use for your system.
+Both databases are fully supported, and you should consider the [pros and cons](https://www.google.com/search?q=mongodb+vs+postgresql) of each to determine which one is best suited to your situation.
 
-<!-- FIXME:TL This sentence implies that all the user has to do is select the adapter and nothing else.
-In reality they have to follow the steps in this guide! -->
+Once you have chosen a database to use you will need to make sure that it is correctly installed and set up, and that you are able to connect to it.
+The instructions below will take you through this process to ensure that you are ready to continue with your Keystone project.
 
-If you're following the [quick start guide](/docs/quick-start/README.md), simply select the appropriate adapter for your database of choice when prompted.
-More information on adapter configuration can be found under the _Setup_ sections.
+> **Tip** Take note of the `connection string` that you use to connect to your database, as you will need to know this to set up your Keystone project.
 
-> **Note:** PostgreSQL requires an additional step to create a database.
+## MongoDB
 
-## Installing [MongoDB](https://www.mongodb.com/)
+### Installation
 
-### MacOS
+#### MacOS
 
 The simplest way to install MongoDB is using [Homebrew](https://brew.sh/).
 Refer the [official guide](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/) for more information.
 
-### Windows
+#### Windows
 
 Follow the [official guide](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/) for installing MongoDB on Windows.
 
-### Other platforms
+#### Other platforms
```

Comment:
Total bikeshed, but I've not been using title case generally. I noticed every other title here is changed to title case except this one... I don't hold strong opinions but like consistency, not just here but with the rest of the docs

Reply:
From our style guide:

- Headings at all levels use [Title Case](https://en.wikipedia.org/wiki/Letter_case#Stylistic_or_specialised_usage)

I'll change this one to `Other Platforms` 👍

Reply:
Wait, I'm confused - #2592 changed all the headings to sentence case 🤔